pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -210,12 +210,12 @@ social:
         -# During Hour of Code season, show signup form and announcements.
         -# When it's not Hour of Code, hide signup form and show announcements and highlights.
         -if ["pre-hoc", "soon-hoc", "actual-hoc"].include?(DCDO.get("hoc_mode", false))
-          .col-md-6
-            =view :signup_form
           .col-md-5
             =view :announcements
           .col-md-1
         -else
+          .col-md-12
```

Comment:
This change ultimately won't be checked in; I just made it to expand the form width so it's easier to work with.

Reply:
Please go ahead and revert this then -- let me know if you want to discuss options for how to make this show up locally but not in your PR.